### ISSUE TITLE
Question about the causality Mask of SASRec

### ISSUE THREAD

**Issue by blueColdSea** [liuqidong07/LLMEmb#2]
请问，我注意到SASRec 并没有加入 causality 的 attention，这是否正常？
代码位于models/SASRec.py
Excuse me, I noticed that SASRec did not include causality's attention. Is this normal?
The code is located in models/SASRec.py
`mha_outputs, _ = self.attention_layers[i](Q, seqs, seqs,)
                                                    #   attn_mask=attention_mask)`

**Comment by liuqidong07** [comment]
您好，感谢对我们工作的关注。这里是因为我试验中发现这样设置sasrec本身的效果会更好。

**Comment by blueColdSea** [comment]
感谢回复！
